pyproject.toml

```diff
@@ -14,6 +14,7 @@ sanic-jwt = "^1.3.2"
 Sanic-Cors = "^0.9.9"
 passlib = {version = "^1.7.1", extras = ["argon2"]}
 cerberus = "^1.3.1"
+python-dotenv = "^0.10.3"
```

Comment:
I added .env in gitignore

Reply:
Is the `.gitignore` file checked in? Also do you mind adding `.env.default` so that people can just copy it into `.env` and then change the values.

Also, do you mind making this a dev dependency? Something like
```
poetry remove python-dotenv
poetry add -D python-dotenv
```
should suffice.